docs/central-install-digital-ocean.rst

```diff
@@ -102,11 +102,27 @@ Getting and Setting Up Central
 
 #. Make sure you are running Docker Engine v23.x and Docker Compose v2.16.x or greater.
 
-.. code-block:: bash
+   .. code-block:: bash
+
+    $ docker --version && docker compose version
+
+   If you are using old versions, follow the instructions to install `Docker Engine <https://docs.docker.com/engine/install/ubuntu>`_ (not Desktop) for Ubuntu, the operating system we recommend and support. The instructions will help you setup the Docker ``apt`` repository and install the latest version of Docker Engine and Docker Compose.
+
+#. Disable the system firewall for web form features to work correctly.
+
+   .. code-block:: bash
+
+    $ ufw disable
+
+   You should see the message ``Firewall stopped and disabled on system startup``. If so, you have configured the firewall correctly.
+
+   .. note::
+
+    While it sounds dangerous, disabling your firewall does not put your server at greater risk. In fact, most Linux operating systems come with the firewall disabled.
 
-  $ docker --version && docker compose version
+    If you don't want to disable the firewall entirely, you can instead configure Docker, ``iptables``, and ``ufw`` yourself. This can be difficult to do correctly, so we don't recommend most people try. Another option is to use an upstream network firewall.
 
-If you are using old versions, follow the instructions to install `Docker Engine <https://docs.docker.com/engine/install/ubuntu>`_ (not Desktop) for Ubuntu, the operating system we recommend and support. The instructions will help you setup the Docker ``apt`` repository and install the latest version of Docker Engine and Docker Compose.
+    The goal here is to ensure that it is possible to access the host through its external IP from within each Docker container. In particular, if you can successfully ``curl`` your Central website over HTTPS on its public domain name.
```

Comment:
The last sentence is missing a clause. Maybe "To verify that this is the case, try to ``curl`` your Central website over HTTPS on its public domain name from within one of the containers.``

Is this even the right thing?